src/en/help-oracle.md

```diff
@@ -0,0 +1,180 @@
+Title: Using the Oracle Compute Provider
+TODO: Test Oracle
+      Rewrite so its not so list heavy (especially Networks and spaces section)
+
+# Using the Oracle Compute Provider
+
+!!! Note:
+	Oracle Compute support is currently in [Juju 2.2 beta3][jujubeta] and
+        is experimental. There may be bugs. Work is continuing on improving the user
+        experience.
+
+To start using Juju with the [Oracle Compute Cloud][compute], some initial set up steps
+are required:
+
+1. Associate Ubuntu or Windows images from the marketplace with your Oracle account.
+   (this is an Oracle legal requirement).
+
+1. If not using a paid account, use `juju add-cloud` to set up the region and
+   endpoint details for Juju to use.  Set up credentials using `juju
+   add-credentials`. 
+
+1. Set up credentials using `juju add-credentials`.
+
+These steps - outlined below - will be refined as we work with Oracle to
+improve the user experience.
+
+Juju and Oracle can be used with either a free Oracle trial account, or a paid
+account.
+
+## Associate images
+
+For each Ubuntu series on which Juju is to run workloads, the relevant image
+needs to be selected from the Marketplace and associated with your account. You
+also need to agree to Oracle’s Terms and Conditions.
+
+The currently available Ubuntu images are:
+
+| Series   | Version                          |
+|----------|----------------------------------|
+| precise  | Ubuntu.12.04-LTS.amd64.20170417  |
+| trusty   | Ubuntu.14.04-LTS.amd64.20170405  |
+| xenial   | Ubuntu.16.04-LTS.amd64.20170330  |
+| zesty	   | Ubuntu.17.04.amd64.20170412.1    |
+
+Windows images are also available for Windows Server 2008 and 2012.
+
+!!! Warning:
+        Do not attempt to associate a `yakkety` image with your account. Only use
+        one of the images listed above.
+
+### Steps to associate images:
+
+Sign in to [https://cloud.oracle.com/home][cloudoracle] with your credentials.
+You will need to know your Data Center and Identity Domain.
+
+1. If you're not on the *Dashboard*, click the `Dashboard` button.
+1. From the Dashboard, click on the green `Create Instance` button.
+1. Select `Compute` from the pop-up windows; you'll come to the Compute screen.
+1. Select `Marketplace` from the list on the left of your screen.
+1. Search for Ubuntu and click `Select` on an image matching one of those listed above.
+1. Accept the terms of use in the 'Install Marketplace Image' pop-up and click `Install`.
+1. Informational messages on install status will appear in the upper right corner.
+1. You can cancel the rest of the image creation at this point.
+
+To verify, go back to the Dashboard. Click on the `Compute` link (one
+of the boxes on your Dashboard). From the 'Service Details: Oracle Compute Cloud
+Service' window, click on `Open Service Console` and select the `Images` tab.
+You'll see a drop down menu; change `Category: All` to `Category: Personal` and
+you should see your image. 
+
+## Set up the Oracle cloud
+
```

Comment:
I agree. But I think we've got to push this now. It would be good to try it out and see what the actual differences are.